src/main/java/com/google/devtools/build/lib/skylarkbuildapi/SkylarkActionFactoryApi.java

```diff
@@ -159,17 +159,70 @@
   @SkylarkCallable(
       name = "symlink",
       doc =
-          "<p><b>Experimental</b>. This parameter is experimental and may change at any "
-              + "time. Please do not depend on it. It may be enabled on an experimental basis by "
-              + "setting <code>--experimental_allow_unresolved_symlinks</code></p><p>"
-              + "Creates a symlink in the file system. If the output file is a regular file, the "
-              + "symlink must point to a file. If the output is an unresolved symlink, a dangling "
-              + "symlink is allowed.",
+          "Creates a symlink in the file system."
+              + "<p>If the output file is a regular file (i.e. created by "
+              + "<a href=\"#declare_file\"><code>declare_file()</code></a>), the symlink must "
+              + "point to a file. If the output is an unresolved symlink (i.e. created by "
+              + "<a href=\"#declare_symlink\"><code>declare_symlink()</code></a>), a dangling "
+              + "symlink is allowed.</p>",
```

Comment:
Suggested rewording:

> This function must be called with exactly one of `target_file` and `target_path` specified.
>
> If `target_file` is used, then `output` must be declared as a regular file (such as by using `declare_file()`). It will actually be created as a symlink that points to the path of `target_file`.
>
> If `target_path` is used instead, then `output` must be declared as a symlink (such as by using `declare_symlink()`). In this case, the symlink will point to whatever the content of `target_path` is. This can be used to create a dangling symlink.